src/panels/lovelace/strategies/original-states-strategy.ts

```diff
@@ -0,0 +1,94 @@
+import { STATE_NOT_RUNNING } from "home-assistant-js-websocket";
+import { subscribeOne } from "../../../common/util/subscribe-one";
+import { subscribeAreaRegistry } from "../../../data/area_registry";
+import { subscribeDeviceRegistry } from "../../../data/device_registry";
+import { subscribeEntityRegistry } from "../../../data/entity_registry";
+import { generateDefaultViewConfig } from "../common/generate-lovelace-config";
+import {
+  LovelaceViewStrategy,
+  LovelaceDashboardStrategy,
+} from "./get-strategy";
+
+let subscribedRegistries = false;
+
+export class OriginalStatesStrategy extends HTMLElement {
```

Comment:
Why make this extend `HTMLElement`?

Reply:
Otherwise we can't register it in the custom elements registry.

Reply:
Yes you can: 
![image](https://user-images.githubusercontent.com/5662298/115782285-c7216b00-a3bb-11eb-98a8-3e1513e946c3.png)